debugclient/main.c

```diff
@@ -7,7 +7,7 @@
    | This source file is subject to version 1.0 of the Xdebug license,    |
    | that is bundled with this package in the file LICENSE, and is        |
    | available at through the world-wide-web at                           |
-   | http://xdebug.derickrethans.nl/license.php                           |
+   | https://xdebug.org/license.php                                       |
```

Comment:
You probably should add your name to the "Authors:" list below too - you nearly rewrote all of it :-) (Same for xdebug_com.c)